### PR TITLE
Integrated the jaeger project bi-weekly calendar into the website #383

### LINKED ISSUE #383: Integrate Jaeger Calendar into the website
In order to bring visibility to our bi-weekly meeting and to encourage people to share their presentations about Jaeger, we could integrate the Jaeger Calendar into the website, possibly under the "Get in touch" page:

https://calendar.google.com/calendar/embed?src=77a1bva4sn9cm822r8oa03l2j0%40group.calendar.google.com

cc @pavolloffay, @objectiser

### PR DESCRIPTION
**Pull request by JennySimen**
Signed-off-by: Yakam Jennifer <yakamjennifer@gmail.com>

 Which problem is this PR solving?
The PR adds the Jaeger project bi-weekly calendar into the website.

 Short description of the changes
It helps to bring visibility to our bi-weekly meeting and resolves #383

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -34,3 +34,4 @@ home = [ "HTML", "JSON", "REDIRECTS" ]
     [[links]]
       title = "Docs"
       url = "/docs"
+theme = "ananke"
```
Comment (yurishkuro) [inline config.toml:37]: ?

**Comment by yurishkuro** [comment]
please make sure all commits are signed. You can do `git commit -s -amend`

**Comment by yurishkuro** [comment]
@JennySimen some of your commits are not signed. https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md#missing-sign-offs

**Comment by JennySimen** [comment]
@yurishkuro thanks will do that

**Comment by JennySimen** [comment]
Hi @yurishkuro thank you for the feedback, I was able to sign-off the unsigned commits. The pr is ready for review again.

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
Thanks!

**Pull request merged**